backend/controllers/prisonerProfile/prisonerQuickLook.js

```diff
@@ -138,15 +138,35 @@ module.exports = ({ prisonerProfileService, prisonApi, logError }) => async (req
         balanceDetails: [
           {
             label: 'Spends',
-            value: formatCurrency((balanceData && balanceData.spends) || 0, balanceData && balanceData.currency),
+            visible: true,
+            html: `<a href='#'>${formatCurrency(
```

Comment:
The urls for each page are within the ACs for each of the individual page.  However I think we should add these as we go so we're not linking to a non existant page.  At the moment, damage obligations is the only one that exists at `prisoner/:offenderNo/prisoner-finance-details/damage-obligations`